py3status/modules/backlight.py

```diff
@@ -101,9 +101,7 @@ def post_config_hook(self):
         if self.device is None:
             raise Exception(STRING_NOT_AVAILABLE)
 
-        self.format = self.py3.update_placeholder_formats(
-            self.format, {'level': ':d'}
-        )
+        self.format = self.py3.update_placeholder_formats(self.format, {'level': ':d'})
```

Comment:
New module `wwan` plus a `backlight` tweak? Ha ha, this shouldn't be here.

Reply:
I don't success to remove this :/